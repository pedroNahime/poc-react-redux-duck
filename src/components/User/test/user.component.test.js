import React from 'react'
import { cleanup, render } from '@testing-library/react'
import {User} from "../user.component";

describe('<User component/>', () => {
    let props;
    beforeEach(() => {
        props = {
            getUser: jest.fn(),
            user:
                {
                    name:{
                        first: 'Helene'
                    }
                }
        }
    });
    afterEach(cleanup);
    test('Render the Component', () =>{
        const {container} = render(<User {...props}/>);
        expect(container).toMatchSnapshot()
    })
});