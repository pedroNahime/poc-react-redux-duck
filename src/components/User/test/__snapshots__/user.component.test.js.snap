// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User component/> Render the Component 1`] = `
<div>
  Helene
</div>
`;
